src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/ResetNullEventTimestampMessageJobTest.java

```diff
@@ -43,15 +49,18 @@ public class ResetNullEventTimestampMessageJobTest {
 
     private final List<String> messageIds = List.of("messageId_1", "messageId_2", "messageId_3");
 
+    private Map<String, Object> caseEventMessageEntityMap = new HashMap<String, Object>();
```

Comment:
I think this is better as a local variable in each method you're using it in instead of here.
Seems like you are only using it to pass a couple of strings to the builder class.